shcomplete/tfdf.py

```diff
@@ -1,11 +1,9 @@
+from __future__ import division
```

Comment:
That's not needed for Python3

Reply:
Ok I understand now, when I ran converage on my computer, It did with python2, that's why I had those mistake. But on github Travis understands that it should "run coverage" with python3